wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -122,6 +122,7 @@ public function get_instance_schema() {
 				'default' => '',
 				'sanitize_callback' => 'sanitize_text_field',
 				'description' => __( 'Title for the widget' ),
+				'should_preview_update' => false,
```

Comment:
If we eventually move to using React or Vue, then this `should_preview_update` would be irrelevant, correct?